packages/twenty-server/src/core/auth/emails/password-update-notify.email.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Link } from '@react-email/components';
+
+import { MainText } from 'src/emails/components/MainText';
+import { Title } from 'src/emails/components/Title';
+import { BaseEmail } from 'src/emails/components/BaseEmail';
+import { CallToAction } from 'src/emails/components/CallToAction';
+import { HighlightedText } from 'src/emails/components/HighlightedText';
+
+type PasswordUpdateNotifyEmailData = {
+    userName: string;
+};
+
+export const PasswordUpdateNotifyEmail = ({
+    userName,
+}: PasswordUpdateNotifyEmailData) => {
+    const helloString = userName?.length > 1 ? `Hello ${userName}` : 'Hello';
+    return (
+        <BaseEmail>
+            <Title value="You changed your password" />
+            <HighlightedText value="Your password changed" />
+            <MainText>
+                {helloString},
+                <br />
+                If you didn't change your password, please contact your workspace administrator right away.
+                <br />
+            </MainText>
+        </BaseEmail>
```

Comment:
> don't see any Figma design for that mail, did you infer it on your onw? If yes I will validate the design and > wording with @Bonapara 

edit: Seen with @Bonapara here is a Figma design for this email https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=20829-56525&mode=design&t=yclxfsOGGY9rOqAL-0
The Object would be: `Your Password Has Been Successfully Changed`
<img width="380" alt="image" src="https://github.com/twentyhq/twenty/assets/29927851/355113a1-8eb0-465a-a77d-7e5e53593249">

Reply:
I didn't find this design in the original issue. Thanks for noting it. I've made necessary changes and now, it should match with the Figma one.